.pre-commit-config.yaml

```
@@ -33,3 +33,8 @@ repos:
       name: format_checkers
       entry: python cve_bin_tool/format_checkers.py
       files: "^cve_bin_tool/checkers/__init__.py"
+
+-   repo: https://github.com/jorisroovers/gitlint
+    rev:  v0.17.0
+    hooks:
+    -   id: gitlint
```

```suggestion
    - id: gitlint
```
To make it consistent with other hooks